lang/LangPrimSource/PyrListPrim.cpp

```diff
@@ -44,6 +44,8 @@ int class_identdict_index, class_identdict_maxsubclassindex;
 PyrClass *class_identdict;
 PyrSymbol *s_proto, *s_parent;
 PyrSymbol *s_delta, *s_dur, *s_stretch;
+PyrSymbol *s_type, *s_rest, *s_empty, *s_r;
+PyrClass *class_rest, *class_metarest;
```

Comment:
I think we should not add these; I've already explained why but I'm happy to discuss it again.